frontend/viewer/src/lib/sandbox/EditorSandbox.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import { EditorGrid, EditorField } from '$lib/components/editor';
+  import MultiSelect from '$lib/components/field-editors/multi-select.svelte';
+  import { ResizablePaneGroup, ResizablePane, ResizableHandle } from '$lib/components/ui/resizable';
+  import { Switch } from '$lib/components/ui/switch';
+  import { Tabs, TabsList, TabsTrigger } from '$lib/components/ui/tabs';
+  import { fieldData } from '$lib/entry-editor/field-data';
+  import { t } from 'svelte-i18n-lingui';
+
+  const allDomains = $state<{label: string}[]>([
+    { label: 'fruit' }, { label: 'tree' }, { label: 'stars' }, { label: 'earth' },
+  ]);
+  for (let i = 0; i < 100; i++) {
+    allDomains.push({
+      label: allDomains[Math.floor(Math.random() * allDomains.length)].label + '-' + i
+    });
+  }
+  allDomains.sort((a, b) => a.label.localeCompare(b.label));
+
+  function randomSemanticDomainSorter() {
+    return Math.random() - 0.5;
+  }
+
+  let selectedDomains = $state([allDomains[0], allDomains[80]]);
+
+  let semanticDomainOrder = $state<'selectionOrder' | 'optionOrder' | 'randomOrder'>('selectionOrder');
+  const sortSemanticDomainValuesBy = $derived(semanticDomainOrder === 'randomOrder'
+    ? randomSemanticDomainSorter
+    : semanticDomainOrder);
+  let semanticDomainsReadonly = $state(false);
+</script>
+
+<ResizablePaneGroup direction="horizontal">
+  <ResizablePane class="!overflow-visible" defaultSize={100}>
+    <div class="@container my-4 border px-4 py-8 relative z-0">
+      <div class="breakpoint-marker w-[32rem] text-orange-600">@lg</div>
+      <div class="breakpoint-marker w-[48rem] text-green-600">@3xl</div>
+      <EditorGrid>
```

Comment:
should we have an editor grid root where the container is defined I wonder?

Reply:
I added an EditorRoot component and explicitly named the container there: `@container/editor`.